dowhy/causal_estimators/causalml.py

```diff
@@ -0,0 +1,108 @@
+import inspect
+import numpy as np
+import pandas as pd
+import pdb
+
+from dowhy.causal_estimator import CausalEstimate
+from dowhy.causal_estimator import CausalEstimator
+from importlib import import_module
+import causalml
+
+class Causalml(CausalEstimator):
+
+    def __init__(self, *args, **kwargs):
+
+        super().__init__(*args, **kwargs)
+
+        # Add the core method used in the estimator
+        self.identifier_method = self._target_estimand.identifier_method
+        self.logger.debug("The identifier method used {}".format(self.identifier_method))
+
+        # Check the backdoor variables being used
+        self.logger.debug("Back-door variables used:" +
+                          ",".join(self._target_estimand.backdoor_variables))
+        
+        # Add the observed confounders and one hot encode the categorical variables

```

Comment:
I wonder if lines 25-50 that set up common causes, effect modifiers and IVs should go in three separate methods in the base causalestimator class, that any new subclass can use.

Method signatures can be, for example: get_instrument_variables_values(), get_effect_modifiers_values(), get_common_causes_values().

Reply:
If this makes sense, you can also edit econml class based on this. 
And no need to call get_iv_values() method from causalml init because causalml has no method that uses IV.

Reply:
It makes sense to leave out the IV variables. However, we want to see if we just run the estimator the results obtained should not be as effective as an informed decision to exclude the instrumental variables. On the other hand, we may see for some estimators, that it may automatically exclude or reduce the influence of the instrumental variables. What do you think?